templates/index.html

```diff
@@ -2,17 +2,97 @@
 {% block title %}To-Do App{% endblock %}
 
 {% block content %}
-  <div class="jumbotron">
-    <h1 class="display-4">To-Do App</h1>
-    <p class="lead">Just another to-do app.</p>
-  </div>
+    <div class="jumbotron">
+        <h1 class="display-4">To-Do App</h1>
+        <p class="lead">Just another to-do app.</p>
+    </div>
 
-  <div class="row">
-    <div class="col-md-12">
-      <h2>Items</h2>
-      <ul class="list-group mb-4">
-        <!-- List the todo items here -->
-      </ul>
+    <div class="row">
+        <div class="col-md-12">
+            <h2>Items</h2>
+            <div class="list-group mb-4 container">
+                {% if items is defined and items %}
+                    {% for item in items %}
+                        <div class="list-group-item d-flex justify-content-between align-items-center row">
+                            <div class="col-8">{{ item.title }}</div>
+                            <div class="col-2">
+                                {% if item.status == 'Not Started' %}
+                                    <form id="complete_item" method="post" action="">
```

Comment:
action seems to not be working properly when setup to be dynamic. I was dynamically creating the uri but the id was being set wrongly. had to extract it into js method and set the action there